lib/puppet/util/windows/user.rb

```diff
@@ -8,13 +8,11 @@ module Puppet::Util::Windows::User
   extend FFI::Library
 
   def admin?
-    elevated_supported = Puppet::Util::Windows::Process.supports_elevated_security?
+    return false unless check_token_membership
```

Comment:
Ended up being pretty convenient that we already had the admin group check code implemented!